tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -361,8 +177,11 @@ tensorflow::Status ConvertGraphDefToTensorRT(
   tensorflow::DeviceProperties device_properties;
   device_properties.set_type("GPU");
   device_properties.mutable_environment()->insert({"architecture", "6"});
-  tensorflow::grappler::Cluster* cluster =
-      new tensorflow::grappler::VirtualCluster({{"/GPU:0", device_properties}});
+  device_properties.set_num_cores(3584);
```

Comment:
Are these for debugging purposes only?

Reply:
Updates to grappler requires num_cores and frequency to be set since they added a check that FATALs if flops calculated using these values is not greater than 0.

Reply:
Would you please add a comment to explain these numbers then?

Reply:
Yes they are. Added comments